src/elements/Container/Container.js

```diff
@@ -5,18 +5,19 @@ import React from 'react'
 import {
   childrenUtils,
   customPropTypes,
-  getElementType,
+  ElementType,
   getUnhandledProps,
   META,
   SUI,
   useKeyOnly,
   useTextAlignProp,
+  withComputedType,
 } from '../../lib'
 
 /**
  * A container limits content to a maximum width.
  */
-function Container(props) {
+const InnerContainer = (props) => {
```

Comment:
1. As `Container` is wrapped with HOC, we can't user `prototype.name`. So, we can use arrow functions for components
2. `_meta` should be exported on the default export, now it's an enhanced component. However, I'm not happy about `Inner-` prefix